intranet/scss/components/_block-comment.scss

```diff
@@ -0,0 +1,56 @@
+.block-comment {
```

Comment:
![bug1](https://cloud.githubusercontent.com/assets/8131755/21413577/e4ea3dcc-c82a-11e6-950e-67a05bf6e7dd.png)

Please check perfect px again. title change to font bold, font size 16px.  font date is normal